meson.build

```diff
@@ -358,6 +361,14 @@ use_ptrace_wrap = use_ptrace_wrap and has_debugger
 message('HAVE_PTRACE: @0@'.format(have_ptrace))
 message('USE_PTRACE_WRAP: @0@'.format(use_ptrace_wrap))
 
+if get_option('install_sigdb')
+  r = run_command(py3_exe, check_meson_subproject_py, 'sigdb')
+  if r.returncode() == 1 and get_option('subprojects_check')
+    error('Subprojects are not updated. Please run `git clean -dxff subprojects/` to delete all local subprojects directories. If you want to compile against current subprojects then set option `subprojects_check=false`.')
+  endif
+  subproject('sigdb', default_options: ['install_dir=@0@'.format(get_option('prefix') / rizin_sigdb)])
```

Comment:
Can the prefix be handled automatically by the subproject?
Something like: `['prefix=@0@'.format(get_option('prefix'), 'install_dir=@0@'.format(rizin_sigdb)]`
Or even better: `['prefix=@0@'.format(get_option('prefix')]` if the sigdb meson knows the right subdirectory itself.

Reply:
this does make sense to just use the prefix and just let the repo to decide exactly in which subfolder.

Reply:
no, it isn't right now

Reply:
i have added rz_sign_DATDIR and i tried manually after installing rizin and does work: https://github.com/rizinorg/sigdb/blob/main/meson.build

Reply:
![image](https://user-images.githubusercontent.com/561184/155823904-f49d8837-95fd-43ea-86d7-15f1e2b8a361.png)
Before and after install